db/schema.rb

```diff
@@ -115,8 +115,8 @@
     t.string "feedburner_orig_link"
     t.string "feedburner_orig_enclosure_link"
     t.boolean "is_perma_link"
-    t.datetime "source_updated_at", precision: nil
     t.string "keyword_xid"
+    t.datetime "source_updated_at", precision: nil
```

Comment:
Extraneous change? (I sometimes just manually undo these, so they're not part of the PR).